documentation/final-report/tex/introduction.tex

```diff
@@ -1 +1,13 @@
-\section{Introduction}
\ No newline at end of file
+\section{Introduction}
+
+During the past 9 weeks, we have worked on designing, implementing and testing a software application. In this report, we will discuss the product and the process that goes with it. We will start by explaining the problem that we tried to solve, and the end-user requirements that we had to take into account.
+
+\subsection*{Problem description}
+The software applicaton we have developed was focused on helping PolyCast, a classical recording company with clients world-wide. PolyCast does a lot of classical concert recordings, but this was stil a very manual operation. It involves paper scripts, manual camera operation and a lot of shouting and chaos during a recording. Because of this, camera operators can't be as efficient as they could be when a more automated work flow would be used. This is where we stepped in, and created a software system that focuses on faster, digital script creation that can be linked with a web application, in which camera operators, score callers and directors all have their own view for during live production. We also integrated automatic preset recalling, so that camera operators can focus more on what's happening now, rather than what is coming later.
```

Comment:
"Polyhymnia International B.V. is a classical recording company with clients world-wide." -> took that directly from their website